content/4_Gold/String_Hashing.mdx

```diff
@@ -100,8 +100,6 @@ export const problems = {
 	''
 ), -->
 
-<FocusProblem problem={problems.ex[0]} />
-
 ## Tutorial
```

Comment:
under Implementation - Single Base, mention the max probability of collision as given in the blog below. Also mention what exactly the single base implementation is calculating.

Reply:
In particular, mention that both two bases and two mods are fine?

Reply:
update the list of authors